### ISSUE TITLE
Dlouhá prodleva načtení okna pro hledání

### ISSUE THREAD

**Issue by nappex** [frenzymadness/donors_registry#153]
**Problém**
při zobrazení soupisu oceněných dárců pro konkrétní medaily je prodleva 3 - 4 sekundy než se mi vykreslí celá stránka tzn. vč vyhledávacího okna... 

Dle mě to bude při častém používáni pro uživatele otravné.

**Replikování problému**
1. Spusť aplikaci
2. otevři link např. [http://127.0.0.1:5000/award_prep/st](http://127.0.0.1:5000/award_prep/st)
3. čekej a sleduj zobrazování stránky

**Comment by frenzymadness** [comment]
Otázka je, kolik lidí najednou v reálu čeká na medaili. Pokud jich je stejně nebo více jako v testovacích datech a rychlost tedy bude problém i na produkci, je možné to řešit jako u přehledové tabulky a načítat data z backendu po částech.

**Comment by nappex** [comment]
Ano, rozumím. Dokonce, si tak matně vzpomínám, že už jsme to řešili. Nedokážu posoudit kolik lidí tam maximálně může být a zda vůbec víme při jakém počtu to začne dělat viditelné potíže. Asi to nemusíme řešit teď, ale issue bych asi nechal a uvidíme v budoucnu co s ní ?
